### PR TITLE
gpavelar/c03-docker03

### PR DESCRIPTION
**Pull request by gpavelar**
# Description

Exercise (e.g. C01-E01): C03-DOCKER03

Student Name: Gustavo Avelar

> Push docker images to Dockerhub.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker03/gpavelar/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+3313c0c04ade: Pushed
+310889822143: Pushed
+beee9f30bc1f: Pushed
+v1: digest: sha256:8d09be6febeb7c234de4c9f52d521dbfaeda1340f9ca87d84501d3f335ee3d13 size: 1569
+
+
+## Command
+docker tag fcf95bce2dd9 gpavelar/exercise02:v2
+# Output
+The push refers to repository [docker.io/gpavelar/exercise02]
```
Comment (alanlima) [inline classes/03class/exercises/c03-docker03/gpavelar/README.md:34]: missing the command to push the image

File: classes/03class/exercises/c03-docker03/gpavelar/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## I've already done another example how to push image to dockerhub, so
+## all my credentials were set up.
+## Docker images
+docker images
+REPOSITORY            TAG                 IMAGE ID            CREATED             SIZE
+exercise02            v2                  fcf95bce2dd9        21 hours ago        107MB
+exercise02            v1                  73a3b15c384b        23 hours ago        107MB
+
+## Tagging image
+docker tag 73a3b15c384b gpavelar/exercise02:v1
```
Comment (alanlima) [inline classes/03class/exercises/c03-docker03/gpavelar/README.md:17]: You don't need to specify the image id hash, you could use the current tag as source `exercise02:v1`.

File: classes/03class/exercises/c03-docker03/gpavelar/README.md (hunk 1 above)
Comment (gpavelar) [reply classes/03class/exercises/c03-docker03/gpavelar/README.md:34]: Sorry about that :/

Comment (gpavelar) [reply classes/03class/exercises/c03-docker03/gpavelar/README.md]: I've just fixed it.

File: classes/03class/exercises/c03-docker03/gpavelar/README.md (hunk 2 above)
Comment (gpavelar) [reply classes/03class/exercises/c03-docker03/gpavelar/README.md:17]: Thanks for your time and review comments  :+1:

File: classes/03class/exercises/c03-docker03/gpavelar/README.md (hunk 1 above)
Comment (alanlima) [reply classes/03class/exercises/c03-docker03/gpavelar/README.md]: Thanks mate 🙂

**Pull request merged**